### DISCUSSION TITLE
A whole bunch of characters appear when updating

### DISCUSSION THREAD

**Discussion by dongrentianyu** [web-infra-dev/rspress#816] [General]
![image](https://github.com/web-infra-dev/rspress/assets/109711900/e88c212d-4256-4e01-8f42-aeaafcefb6b3)

如图所示。因为之前更新的时候不会出现一大串`(node:12128) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 close listeners added to [TLSSocket]. Use emitter.setMaxListeners() to increase limit`代码。

我不是程序员，不清楚这是因为我的电脑问题，还是更新就是这样的。

**Comment by dongrentianyu** [comment]
![image](https://github.com/web-infra-dev/rspress/assets/109711900/ae27c4a7-9db9-44f9-a0b9-931e902fb655)

最终能够正常更新。所以我也不确定是不是有问题。

**Reply by sanyuan0704** [reply]
看安装结果是成功的，这个 node 的报错可以重启电脑看看

**Reply by dongrentianyu** [reply]
好的，谢谢解答

**Comment by 10Derozan** [comment]
> 好的，谢谢解答

Have you solved this problem?

**Reply by dongrentianyu** [reply]
No, I haven't found a solution yet.

**Comment by dongrentianyu** [comment]
重新启动甚至重新更新了电脑，也没有解决。node是v20.9.0，npm是10.2.3。网上也没有搜索到答案。
